AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -185,6 +186,11 @@
     // ETA
     private int eta;
 
+    // JavaScript Versioning
+    protected String apiVersion = "";
```

Comment:
Could this follow the naming standards

```suggestion
    protected String mCardSuppliedApiVersion = "";
```